component_lib/src/components/FileDatabase/template.html

```diff
@@ -25,11 +25,11 @@
       </v-toolbar>
       <v-divider />
       <v-list>
-        <v-list-item-group :value="selectedFileIndex">
+        <v-list-item-group :value="value ? value.id : ''" @change="selectFile">
```

Comment:
You changed my mind. I now think the `@change` here makes sense because we have a method to call instead of just updating `value`.